serverless.yml

```diff
@@ -1,14 +1,21 @@
 service: scale-of-belief-import
+frameworkVersion: '3'
+configValidationMode: error
+plugins:
+  - serverless-webpack
+  - serverless-offline
 
 provider:
   name: aws
   stage: ${env:ENVIRONMENT}
-  runtime: nodejs10.x
   region: us-east-1
   vpc:
     securityGroupIds:
-      - ${file(${env:ECS_CONFIG}/bin/vars.yml):us-east-1.webapp_elb_sg}
-    subnetIds: ${file(${env:ECS_CONFIG}/bin/vars.yml):us-east-1.prod_apps_1_all}
+      - sg-888294ed
+    subnetIds:
+      - subnet-f2c2dfda # us-east-1a
+      - subnet-1508db62 # us-east-1b
+      - subnet-3fd94266 # us-east-1c
   stackTags:
     Name: ${env:PROJECT_NAME}-${self:custom.environmentMap.${env:ENVIRONMENT}}
```

Comment:
This is the only place I think it is used.

Reply:
yup, and that pulls it from the environment and not NodeJS env. the `.env` is really just what needs to get passed along from the environment to the lambda environment.